rfcs/convergence/popper-js.md

```diff
@@ -0,0 +1,136 @@
+- CSS will not flip and reposition based on available viewport space
+- Cannot follow mouse cursor or be used as a context menu
+- Hard to modify the behaviour at runtime
+
+Therefore, most solutions are writting in complex javascript that requires handling of all the boundary and scroll cases
+at runtime. This results in very complicated code, that needs to be actively maintained and improved. Looking
+[at the current v7 documentation of Callout](https://developer.microsoft.com/en-us/fluentui#/controls/web/callout), there are
+more than 20 issues that stretch back to 2018 that still have not been resolved.
+
+Popper references and supports the following selection of v8 open issues (following the thread discussions):
+
+- [#7970](https://github.com/microsoft/fluentui/issues/7970) - Beak positioning and offset
+- [#9249](https://github.com/microsoft/fluentui/issues/9249) - Modifiers to create sane resizing/repositioning variants of popper
+- [#5148](https://github.com/microsoft/fluentui/issues/5148) - Decoupled placement properties for each access for easy RTL modifiers
+
+Convergence is moving fast, spending time to build a fully-fledged positioning library that handles all the boundary cases
+can be time consuming and slow down or block development for some components
+
+## Detailed Design or Proposal
+
+<!-- This is the bulk of the RFC. Explain the proposal or design in enough detail for the inteded audience to understand. -->
+
+The important part of this RFC is to get quorum on using popper in Fluent. This will allow us to offload the heavy effort
+of managing out of order DOM popoverse and focus on delivering value to our partners through usable components.
+
+As mentioned above, out of the box popper provides support and configuration for the following scenarios:
+
+- Preventing overflow on available space and window resize
+- Content repositioning on window scroll
+- Content flipping based on available space
+- Content offset from anchor element
+- Beak/Arrow positioning
+
+In the short/long-term future, it is always possible for the team to reevaluate the scenarios supported by popper to see
+if the the library needs to be extended through [popper modifiers](https://popper.js.org/docs/v2/modifiers/) or a custom
+solution, if there is a need for it based on feedback from our partners.
+
+This RFC does not specify exactly how popper should be integrated into fluent, and is left to a technical discussion or
+follow up RFC. Since Popper is independent of any rendering framework (i.e. React), multiple integrations can be proposed
+and we can choose the one best suited for Fluent.
+
+How the library is used can therefore be seen as a technical detail. It can be possible to use popper as an external
+component or hook or JS helper function. Lazy loading of the code module is also possible.
+
+### Pros and Cons
+
+#### Pros
+
+- Covers more scenarios than current v7 Callout
+- Already used by v0 and teams
```

Comment:
There is a reflected con here that none of the other products use popper, but v7 Callout.